### DISCUSSION TITLE
Настройка для gitLab CI

### DISCUSSION THREAD

**Discussion by Leontievna** [qa-guru/allure-notifications#70] [Ideas]
Если хотите подключить chart нужен отдельный image с openJdk который распознает шрифты. 
У меня была такая ошибка :
 _Caused by: java.lang.NullPointerException: Cannot load from short array because "sun.awt.FontConfiguration.head" is null_
Я использовала для этой джобы [dquintela/openjdk-8-jdk-alpine](https://hub.docker.com/r/dquintela/openjdk-8-jdk-alpine). Мне хватило.

**Comment by Tasza33** [comment]
Спасибо!
